assets/sass/components/global/_googlesitekit-publisher-wins.scss

```diff
@@ -203,6 +201,40 @@
 		}
 	}
 
+	.googlesitekit-button-icon--spinner + .googlesitekit-cta-link {
```

Comment:
I think we can probably scope this to `.googlesitekit-publisher-win__actions` rather than the more fragile `+` selector. We could potentially add primary/secondary classes there as well to make it a bit more explicit.